core/store/orm/orm.go

```diff
@@ -440,12 +440,15 @@ func (orm *ORM) Jobs(cb func(*models.JobSpec) bool, initrTypes ...string) error
 		}
 
 		jobs := []models.JobSpec{}
-		err = orm.preloadJobs().Find(&jobs, "id IN (?)", ids).Error
+		err = orm.preloadJobs().Unscoped().Find(&jobs, "id IN (?)", ids).Error
 		if err != nil {
 			return 0, err
 		}
 		for _, j := range jobs {
 			temp := j
```

Comment:
btw this pattern is only needed if the variable is used outside of the loop (think concurrency.  Its not really needed here.

Reply:
The temp pattern? We're passing a pointer to `temp` below to the callback function `cb`. If we didn't assign `j` to a new `temp` variable all invocations of `cb` would receive the same pointer address for `j`. Without knowing what `cb` does, or how it could change in the future, I think it's safer to leave this as is.